reana_workflow_controller/rest.py

```diff
@@ -1330,15 +1330,15 @@ def run_cwl_workflow_from_spec_endpoint(workflow):  # noqa
         abort(400)
 
 
-def run_serial_workflow_from_spec(workflow, engine_parameters):
+def run_serial_workflow_from_spec(workflow, operational_parameters):
     """Run a serial workflow."""
     try:
         kwargs = {
             "workflow_uuid": str(workflow.id_),
             "workflow_workspace": workflow.get_workspace(),
-            "workflow_json": workflow.specification,
-            "workflow_parameters": workflow.parameters,
-            "engine_parameters": engine_parameters,
```

Comment:
If this changes, a PR in each engine is needed.

Reply:
The same things are passed, only the renaming from `engine_...` -> `operational_...` affects `rwe-serial`, PR is ready.